docs/contributing/plugins/index.md

```diff
@@ -1,20 +1,25 @@
----
-sidebar_label: Plugins
----
+# Plugins
```

Comment:
Need to remove this heading, restore the above frontmatter, and move the import below the "Contributing to Plugins" heading below.